### PR TITLE
Change `jsonschemadoc` function to output a title and add a class to the parent div

### PR DESCRIPTION
**Pull request by ryan-blunden**
The `jsonschemadoc` function now outputs a title and adds a class name to the code parent element. Required for https://github.com/sourcegraph/sourcegraph/pull/5133

## Before

Screen Shot 2019-08-08 at 12 52 47 PM

## After

Screen Shot 2019-08-08 at 12 54 34 PM

### CODE REVIEW

**Comment by felixfbecker** [comment]
I am not the right person to review this

File: funcs.go [hunk 1]
```diff
@@ -38,11 +41,13 @@ func createMarkdownFuncs(site *Site) markdown.FuncMap {
 				return "", err
 			}
 
-			var schema *jsonschema.Schema
+			var _, schema *jsonschema.Schema
```
Comment (sqs) [inline funcs.go]: remove `_`

**Comment by ryan-blunden** [comment]
> I recommend adding a test

Not that this should be the norm, but right now, I'd prefer adding tests at a later date as the change is relatively low risk.

**Comment by ijt** [comment]
Can you show before and after screenshots in the description?

**Pull request merged**